blivet/devices/stratis.py

```
@@ -48,7 +48,7 @@ class StratisPoolDevice(ContainerDevice):
 
     _type = "stratis pool"
     _resizable = False
-    _packages = ["stratisd", "stratis-cli"]
+    _packages = ["stratisd", "stratis-cli", "stratisd-dracut"]
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)